runtime/lua/vim/lsp/completion.lua

```diff
@@ -671,6 +671,107 @@ local function get_augroup(bufnr)
   return string.format('nvim.lsp.completion_%d', bufnr)
 end
 
+---@param winid integer
+---@param bufnr integer
+---@param ft? string
+local function update_popup_window(winid, bufnr, ft)
+  if winid and api.nvim_win_is_valid(winid) and bufnr and api.nvim_buf_is_valid(bufnr) then
+    vim.wo[winid].conceallevel = 2
```

Comment:
Consider a condition to `ft` below (to reduce triggering `OptionSet`) or using `eventignorewin` here if it make sense.